apply.yml

```diff
@@ -2,9 +2,10 @@
 - name: Deploy {{ target }} 
   hosts: "{{ target }}"
   vars:
-    ci_cd_namespace: <YOUR_NAME>-ci-cd
-    dev_namespace: <YOUR_NAME>-dev
-    test_namespace: <YOUR_NAME>-test
+    namespace_prefix: "{{ <YOUR_NAME> }}"
```

Comment:
Probably should have no `{{` and `}}` for the `namespace_prefix` (?)

Reply:
Fixed.